third_party/jemalloc.BUILD

```diff
@@ -25,6 +25,8 @@ JEMALLOC_BUILD_COMMAND = """
   export CC=$$(absolutize $(CC))
   export CXX=$$(absolutize $(CC))
   [ "$$(uname)" = "Linux" ] && export AR=$$(absolutize $(AR))
+  [ "$$(uname)" = "Darwin" ] && export DEVELOPER_DIR=$$(xcode-select --print-path)
+  [ "$$(uname)" = "Darwin" ] && export SDKROOT=$$(xcrun --show-sdk-path)
```

Comment:
wow, does this mean that jemalloc now builds on macOS? i have never been able to get that working since like macOS Mohave